Preparation to accommodate more languages

pjanaya opened the pull request:
Changed styles after the feedback from #58 to accommodate the new translations that are going to come in the upcoming days.
Also removed an old script after feedback from #62.

This is an example with fake languages:
<img width="357" alt="Screen Shot 2020-04-15 at 23 07 53" src="https://user-images.githubusercontent.com/5021038/79397862-b3545500-7f6e-11ea-8e18-9bb26a8c9c72.png">

eirikurn commented on src/locales/en.json:
```diff
@@ -1,5 +1,7 @@
 {
   "translation": {
+    "description": "Help the Civil Protection Team to track potential COVID-19 infections in Iceland",
+    "continue": "Continue in English",
```
Let's prefix these with eg "language".

pjanaya replied:
Done!

eirikurn commented on src/router/logged-out/screens/LanguageScreen.tsx:
```diff
@@ -0,0 +1,40 @@
+  };
+}
+
+const LanguageScreen = ({ navigation }: Props) => {
```
Nice converting the screen to tax! :)

eirikurn commented:
Also. We should consider the ordering and presentation of the languages. UX wise I'd like Icelandic to be pre-selected at the top.

But the other languages should probably be ordered alphabetically. Looking at covid.is, they order the languages alphabetically. They also prefix the non-Latin languages (like Thai) with the language name in Icelandic ... which is a bit weird to me, but at least it simplifies the alphabetic ordering.

P.s. awesome job with this PR!

pjanaya commented:
Merged master into this branch, added the missing keys to FR and TH, and changed the order of the languages.

This is how it looks:
![Simulator Screen Shot - iPhone 11 - 2020-04-18 at 00 30 30](https://user-images.githubusercontent.com/5021038/79623552-7aea7d80-810c-11ea-8998-1298e50d8588.png)

eirikurn commented on src/i18n/index.ts:
```diff
@@ -26,7 +26,13 @@ const resources = {
+  flag: string;
+}
+
+export const languages: LanguageDefinition[] = [
```
Fancy sort function...

I would have also accepted just manually ordering this array ;)

pjanaya replied:
This way nobody needs to remember adding new languages in a specific order. Those things tend to be forgotten over time :)

eirikurn commented on src/locales/fr.json:
```diff
@@ -81,6 +83,25 @@
-    "worriesAndAnxietyLink": "https://www.covid.is/sub-categories/worries-and-anxiety"
+    "worriesAndAnxietyLink": "https://www.covid.is/sub-categories/worries-and-anxiety",
+    "links": {
+      "primary": [
```
Good to have this explicit here. 👍

eirikurn commented on src/i18n/index.ts:
```diff
@@ -87,7 +86,7 @@ export default function initI18n() {
       },
       ns: namespace,
-      defaultNs: namespace,
+      defaultNS: namespace,
```
Good catch 😮

pjanaya replied:
Typescript doing its magic 😍
